ios/Plugin/CapacitorCalendar.swift

```diff
@@ -472,6 +498,38 @@ public class CapacitorCalendar: NSObject, EKEventEditViewDelegate, EKCalendarCho
         }
     }
 
+    public func deleteEventById(id: String, span: EKSpan) async throws -> String?
+    {
+        await withCheckedContinuation { continuation in
+
+            var deletedEvent: String?
+
+            guard let event = eventStore.event(withIdentifier: "\(id)") else {
+                return
+            }
+
+            do
+            {
+                try eventStore.remove(event, span: .futureEvents, commit: false)
```

Comment:
This method takes `span` as argument but it always deletes all future events instead of taking `span` into account.

Reply:
I've tried to fix this for my colleague. Sadly I cannot check if it is correct or if it works as I don't have the means to test it.